manifests/m/Microsoft/VFSforGit/1.0.21229.1/Microsoft.VFSforGit.yaml

```diff
@@ -0,0 +1,21 @@
+PackageIdentifier: Microsoft.VFSforGit
+PackageVersion: 1.0.21229.1
+PackageName: VFS for Git
```

Comment:
```suggestion
PackageName: GVFS
```

Reply:
Windows Package Manager requires the name (without the version) to match what's shown in Control Panel.

Reply:
Our half-baked rename continues to cause problems. This is probably fine, though, so I will commit this change. Thanks!